.env.local.example

```diff
@@ -7,3 +7,10 @@ SUPABASE_SERVICE_ROLE_KEY=
 
 # Ollama
 NEXT_PUBLIC_OLLAMA_URL=http://localhost:11434 # Default
+
+# API Keys (Optional: entering an API key here allows users to chat without entering their own key)
+OPENAI_API_KEY=
```

Comment:
dont forget to add `NEXT_` prefix so the envs could be detected in client side.

Reply:
I think this could be unsafe [(src)](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)? I made another route in the API to fetch if a key exists. see 1e67b90cabecb45b58192b43b13ebae750c20a96